corehq/apps/cleanup/management/commands/generate_form_case_consistency_list.py

```diff
@@ -1,161 +0,0 @@
-from datetime import datetime, timedelta
```

Comment:
same with this one. now that we have atomic submissions i think this is deprecated. also i think https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cleanup/management/commands/rebuild_cases_with_mismatched_forms.py sort of serves its original purpose. cc @dannyroberts @snopoke